site/app/templates/admin/UploadWrapperForm.twig

```diff
@@ -2,26 +2,11 @@
 	<h1>Customize Website Theme</h1>
 	<div>
         Instructors can upload simple html files to use as a wrapper around the Submitty website.<br>
```

Comment:
Instructors can no longer upload HTML files. I believe the entire description needs a re-write to note that JSON files are used for sidebar links and CSS files are used for course website styling.

Reply:
im going to delete more of it because you are write that the feature is gone but the re-write is going to be decided by barb.